doc/SwaggerApi.xml

```diff
@@ -0,0 +1,165 @@
+        - user
+      responses:
+        "200":
+          description: 'User id for an app to use in future calls'
+          schema:
+            $ref: '#/definitions/UserId'
+  /accounts/details:
+    get:
+      summary: 'Get basic info about user'
+      operationId: getUserDetails
+      tags:
+        - user
+      parameters:
+      - in: body
+        name: "user id"
+        description: user id
+        required: true
+        schema:
+            $ref: '#/definitions/UserId'
+      responses:
+        '200':
+          description: 'User data'
+          schema:
+            $ref: '#/definitions/UserDetails'
+        '403':
+          description: Bad user id
+  /auctions:
+    get:
+      summary: 'Get latest auctions'
+      operationId: getAuctions
+      tags:
+        - auction
+      parameters:
+      - in: "body"
+        name: "userId"
+        schema:
+          $ref: '#/definitions/UserId'
+      responses:
+        '200':
+          description: 'Auction list retrieved successfully'
+          schema:
+            $ref: '#/definitions/AuctionList'
+        '403':
+          description: Bad user id
+  /auctions/{auctionId}:
+    get:
+      summary: 'Get detailed state of auction'
+      operationId: getAuctionDetails
+      tags:
+        - auction
+      parameters:
+      - in: "path"
+        required: true
+        name: "auctionId"
+        type: integer
+      - in: "body"
+        name: "userId"
+        schema:
+          $ref: '#/definitions/UserId'
+      responses:
+        '200':
+          description: 'User retrieved successfully'
+          schema:
+            $ref: '#/definitions/AuctionDetails'
+        '403':
+          description: Bad user id
+    post:
+      summary: 'Bet on auction'
+      operationId: betAuction
+      tags:
+        - auction
+      parameters:
+      - in: "path"
+        required: true
+        name: "auctionId"
+        type: integer
+      - in: "body"
+        name: "auctionBet"
+        schema:
+          $ref: '#/definitions/AuctionBet'
+      responses:
+        '200':
+          description: 'Bet successful'
+          schema:
+            $ref: '#/definitions/AuctionDetails'
+        '400':
+          description: Not enough value in bet
+        '403':
+          description: Bad user id
+        '404':
+          description: Auction already ended
+definitions:
+  UserId:
+      type: object
+      properties:
+        userId:
+          example: "729b8942-b579-11eb-8529-0242ac130003"
+          type: string
+  UserDetails:
+      type: object
+      properties:
+        cash:
+          type: integer
+        password:
+          type: string
+        ownedMovies:
+          type: array
+          items:
+            type: object
+            properties:
+              name:
+                type: string
+              profit:
+                description: "Movie income - movie price. May be lower than 0"
+                type: integer
+              ownEnd:
```

Comment:
Jakieś pomysły na lepszą nazwe niż ownEnd?

Reply:
Added the description. This is the date from which user will no longer own this video. We need this to detect how this video should be displayed in the list.

I thought we will show the list of all videos that user ever owned and we will display the videos that he owned but no longer own with grey backgorund of sth like that.

Reply:
I would went with videoExpiryDate or , expiryDate

Reply:
Followed your idea but ended with rentExpirationDate. Should be ok.